libc/string.c

```diff
@@ -26,16 +26,13 @@
 
 
 /* fixtures for strlcpy */
-char source[SRC_SIZE];
-char dest[DEST_SIZE];
+static char source[SRC_SIZE];
```

Comment:
@damianloew @niewim19 IMHO small buffers should be allocated on stack in the functions using them (no need to increase total bss size of the binary as we're using them).

Also: common `TEST_SETUP` should set up only things that are common for all tests, not which might eventually be used by some test (local variables will solve it partially).